SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialMetricsDisplay.cpp

```diff
@@ -143,7 +159,7 @@ void ASpatialMetricsDisplay::Tick(float DeltaSeconds)
 {
 	Super::Tick(DeltaSeconds);
 
-	if (!GetWorld()->IsServer())
+	if (!GetWorld()->IsServer() || !HasActorBegunPlay())
```

Comment:
ugh I didn't realize the engine would call Tick() before BeginPlay() :(

Reply:
I was caught out by it too. It's only because this extends from AInfo, where `bAllowTickBeforeBeginPlay` is set to true.